lib/CommentCompilationWarning.js

```diff
@@ -6,7 +6,25 @@
 
 const WebpackError = require("./WebpackError");
 
+/** @typedef {import("./Module.js")} Module */
+
+/** @typedef {Object} Position
+ *  @property {number} column
+ *  @property {number} line
+ */
+
+/** @typedef {Object} Loc
```

Comment:
Should I add ` @property {number=} index` like in compareLocations.js ?

Reply:
@mohsen1  The `Loc` type I have defined here is used in several locations, is there a way to not `typedef` it over and over again, but just import it from a common definition file?

Reply:
You can import typedefs from other files. Maybe the best place is in ``Dependency.js`

Reply:
@ooflorent Can you make a comment on sokras comment?

Reply:
@mohsen1 ?

Reply:
Importing types from  Dependency class makes sense.

Reply:
Please review the changes i made, advise appreciated. I have commited this change, but I have TS errors

![image](https://user-images.githubusercontent.com/966368/41729565-ce99d5f2-7579-11e8-9436-7253c0590405.png)

Actually, I have no clue how to do this ;-)